<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>961</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>961</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="main_chat_btn">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>450</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>자유 채팅방</string>
     </property>
    </widget>
    <widget class="QPushButton" name="main_login_btn">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>210</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="main_sign_in_btn">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>210</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_login_label">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>240</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>####</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fu_game_move">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>450</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>기태 게임</string>
     </property>
    </widget>
    <widget class="QPushButton" name="main_game2_btn">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>450</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>현도 게임</string>
     </property>
    </widget>
    <widget class="QLabel" name="login_label2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>510</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>###</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QPushButton" name="show_main1">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>500</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>메인</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>250</y>
       <width>321</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QPushButton" name="login_btn">
        <property name="text">
         <string>로그인</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="login_id_edit">
        <property name="text">
         <string>이현도</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>비밀번호</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="login_password_edit">
        <property name="text">
         <string>123</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>아이디</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="login_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_7">
    <widget class="QPushButton" name="show_main2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>500</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>메인</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>250</y>
       <width>401</width>
       <height>126</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLineEdit" name="id_edit">
        <property name="text">
         <string>이현도</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>비밀번호</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="password1_edit">
        <property name="text">
         <string>123</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>비밀번호 재확인</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>아이디</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="sign_btn">
        <property name="text">
         <string>가입하기</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="id_same_btn">
        <property name="text">
         <string>아이디 중복확인</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="password2_edit">
        <property name="text">
         <string>123</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="id_same_label">
        <property name="text">
         <string>아이디 중복 NG</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="sign_label">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>접속 인원</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>자유채팅방</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>250</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>채팅방 목록</string>
     </property>
    </widget>
    <widget class="QPushButton" name="room_add_btn">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>360</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>추가</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="text_edit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>610</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="connect_person_list">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>150</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="chat_room_add_edit">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>360</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>chat_b</string>
     </property>
    </widget>
    <widget class="QPushButton" name="send_btn">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>610</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>보내기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="show_main3">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>670</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>메인</string>
     </property>
    </widget>
    <widget class="QListWidget" name="chat_room_list">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>270</y>
       <width>381</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="chat_view_list">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>410</y>
       <width>381</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="test_btn2">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>채팅방 조회</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QPushButton" name="join_game_btn">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>650</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>입장</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>접속 인원</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="game_lineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>620</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="message_send_btn">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>620</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>보내기</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>490</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>게임 입장 전 채팅방</string>
     </property>
    </widget>
    <widget class="QPushButton" name="client_create_rm_btn">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>650</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>방만들기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="show_main4">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>660</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>메인</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>510</y>
       <width>381</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>391</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_4">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>60</y>
       <width>561</width>
       <height>551</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>방 번호</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>방 제목</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>현재 인원</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>최대 인원</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>게임 종류</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="invite_accept_btn">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>850</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>수락</string>
     </property>
    </widget>
    <widget class="QLabel" name="invite_message_label">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>710</y>
       <width>331</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>게임방에서 초대가 왔습니다.  수락하시겠습니까?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="invite_reject_btn">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>850</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>거절</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QListWidget" name="connect_person_list2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>150</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>공룡게임</string>
     </property>
    </widget>
    <widget class="QListWidget" name="connect_person_list3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>240</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="attend_label">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>160</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>##</string>
     </property>
    </widget>
    <widget class="QPushButton" name="show_main5">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>670</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>메인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="game_start_btn">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>270</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>게임 시작</string>
     </property>
    </widget>
    <widget class="QPushButton" name="game_chat_btn">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>550</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>보내기</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="text_edit2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>550</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>220</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>게임 참가 인원</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>130</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>접속 인원</string>
     </property>
    </widget>
    <widget class="QListWidget" name="chat_view_list2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>330</y>
       <width>381</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="attend_btn">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>180</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>참가요청</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>끝말 잇기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="show_main6">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>690</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>메인</string>
     </property>
    </widget>
    <widget class="QLabel" name="follow_up_word_lbl">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>581</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>72</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:72pt;&quot;&gt;제시어&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="user_1_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255);
color:black;
border-radius:5px;
</string>
     </property>
     <property name="text">
      <string>User_1</string>
     </property>
    </widget>
    <widget class="QLabel" name="user_2_label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>260</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255);
color:black;
border-radius:5px;
</string>
     </property>
     <property name="text">
      <string>User_2</string>
     </property>
    </widget>
    <widget class="QLabel" name="user_3_label">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>260</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255);
color:black;
border-radius:5px;
</string>
     </property>
     <property name="text">
      <string>User_3</string>
     </property>
    </widget>
    <widget class="QLabel" name="user_4_label">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>260</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255);
color:black;
border-radius:5px;
</string>
     </property>
     <property name="text">
      <string>User_4</string>
     </property>
    </widget>
    <widget class="QListWidget" name="follow_up_list_widget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>460</y>
       <width>821</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="follow_up_send_lineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>650</y>
       <width>671</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="follow_up_send_btn">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>650</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>전송</string>
     </property>
    </widget>
    <widget class="QPushButton" name="follow_up_start_btn">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>460</y>
       <width>121</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string>시작</string>
     </property>
    </widget>
    <widget class="QPushButton" name="follow_up_invite_btn">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>260</y>
       <width>121</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string>초대하기</string>
     </property>
    </widget>
    <widget class="QLabel" name="follow_up_rm_number">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>25</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>현재인원</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>최대 인원</string>
     </property>
    </widget>
    <widget class="QLabel" name="current_people_lbl">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>현재인원</string>
     </property>
    </widget>
    <widget class="QLabel" name="max_people_lbl">
     <property name="geometry">
      <rect>
       <x>867</x>
       <y>64</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>4명</string>
     </property>
    </widget>
    <widget class="QLabel" name="Turn_lbl">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="Turn_lbl_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>턴</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>내 ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="client_id_lbl">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
